todo-app-redux/src/app/styles/normalize-antd.scss

```diff
@@ -0,0 +1,11 @@
+// Лоадер на весь контейнер
```

Comment:
I'm not sure we need this comment here at all, but if you think we do, please use English